views/teams/team.hbs

```diff
@@ -305,13 +305,13 @@
 {{#embed "lib/components/modal-form" class="create-event-modal" action=../createEventAction collapseId=1
 userId=../userId teamId=../teamId}}
     {{#content "fields"}}
-    {{> "calendar/forms/form-create-event" showVideoconferenceOption=@root.showVideoconferenceOption}}
+    {{> "calendar/forms/form-create-event" showVideoconferenceOption=@root.showVideoconferenceOption testIdPrefix="create"}}
```

Comment:
Is this the correct way to add an attribute to an HTML element?

Reply:
`calendar/forms/form-create-event` component used twice once for creating an event and the second one is used for editing the event in calendar and teams modules. To make it dynamic I used this to suffix to the relevant component i.e. create and edit